### DISCUSSION TITLE
How to change OpenAI voice in CLI?

### DISCUSSION THREAD

**Discussion by BarrenWardo** [souzatharsis/podcastfy#60] [Q&A]
Is it possible? if not, can it be added?

**Comment by souzatharsis** [comment]
Right now you would need to update it in config.yaml which is not available at CLI/package user interface, only if you clone the project and modify it manually.

It's trivial to make an update such that users can config via CLI/package interface. This will also solve #10 and #6. 
We'll do in the next couple of days.

**Comment by souzatharsis** [comment]
@BarrenWardo See v0.2.3 release. User now can provide TTS specific config such as voices.
